src/services/getDappMetadata.js

```diff
@@ -36,8 +35,12 @@ export default async function getDappMetadata(skylink) {
   const skynetMetadata = await getSkynetMetadata(skylink);
 
   try {
-    const skylinkUrl = await skynetClient.getSkylinkUrl(skylink, { subdomain: true });
-    const contentType = (await ky.head(skylinkUrl)).headers.get("content-type");
+    // grab skylinkUrl for use in fetching metadata.
+    // Doesn't use subdomain:true to ensure JWT cookie gets sent
+    const skylinkUrl = await skynetClient.getSkylinkUrl(skylink);
```

Comment:
I could maybe leave this as a subdomain actually, but open to discussion.

Reply:
I think the JWT should be valid for all subdomains actually. Should double-check with @kwypchlo . In that case we'd want to remove the comment about `subdomain:true`.

Reply:
Do we even need the credentials for getting metadata? Does fetching metadata count as a download in the user's account?

Reply:
Removed the comment and restored subdomain:true in following commit.

We could probably use a different technique to get the content-type of the main file, but HEAD request by the browser is what it is currently.